src/Security/Member.php

```diff
@@ -382,7 +382,7 @@ public function canLogIn()
      *
      * @return bool
      */
-    public function isLockedOut()
+    protected function isLockedOut()
```

Comment:
As per @dhensby his comment, Authenticators should call `canLogin`, which in turn calls `isLockedOut`. Thus it makes sense to make isLockedOut protected and have it as an internal method instead of a public one.
Hence the change to protected